Week2/akif.js

```diff
@@ -0,0 +1,108 @@
+// Part 1 Hello World
+console.log('Hello World'); // English
+console.log('Merhaba Dunya'); // Turkish
+console.log('Hallo Wereld'); // Dutch
+console.log('مرحبا بالعالم'); // Arabic
+
+// Part 2 Error  -----------------------------------------
+console.log("I'm awesome");
+
+// Part 3 Console.log(x)   --------------------------------
+let x;
+console.log('the value of my variable x will be: numberOfTheStudents');
+console.log(x);
+x = 16;
+console.log('the value of my variable x will be: numberOfTheStudents');
+console.log(x);
+
+// Part 4  String y  ---------------------------------------
+let y = 'I love you JavaScript';
+console.log('the value of my variable y will be: feelingAboutJavaScript');
+console.log(y);
+y = 'I will learn you';
+console.log('the value of my variable y will be: myPlanAboutJavaScript');
+console.log(y);
+
+// Part 5  Round a number  ---------------------------------
+let z = 7.25;
+console.log(z);
+let a = Math.round(z);
+console.log(a);
+let b = Math.max(z, a);
+console.log(b);
+
+// Part 6 Arrays   ------------------------------------------
+let myNumbers = [];
+console.log('the value of my variable y will be: myLuckyNumbers');
+console.log(myNumbers);
+let myFavouriteAnimals = ['cat', 'butterfly', 'bear'];
+console.log(myFavouriteAnimals);
+myFavouriteAnimals.push('baby pig');
+console.log(myFavouriteAnimals);
+
+// Part 7  More Strings - Length -----------------------------
+let myString =
+  'JavaScript (JS) is a lightweight interpreted or JIT-compiled programming language with first-class functions.';
+console.log(myString);
+console.log(myString.length);
+
+// Part 8  typeof  -------------------------------------------
+let k = 42;
+let l = 'Hello World';
+let m;
+let n = true;
+console.log(k, l, m, n);
+console.log('the types of my variable accordingly are: number, string, undefined, boolean');
+console.log(typeof k, typeof l, typeof m, typeof n);
+if (typeof k != typeof l) {
```

Comment:
Ah. If you remember from our class last week, we should always strive for using strict equality checks i.e `===` or `!==` instead of `==` or `!=` respectively. Can you please update this?